python/paddle/v2/framework/layers.py

```diff
@@ -35,7 +35,10 @@ def fc(input,
                 "Y": w,
             },
             outputs={"Out": tmp},
-            attrs={'x_num_col_dims': num_flatten_dims})
+            attrs={
+                'x_num_col_dims': num_flatten_dims,
```

Comment:
`x_num_col_dims` and `num_flatten_dims` refer to the same thing. Maybe we should use the same name?

Reply:
`x_num_col_dims` is an attribute of `mul` op, while `num_flatten_dims` is an attribute of `fc` layer. They are at different levels.